portal-api/src/v1/controllers/feedback.controller.js

```diff
@@ -38,18 +36,15 @@ exports.create = async (req, res) => {
 
   const modifiedFeedback = {
     ...req.body,
-    submittedBy: req.user.username,
-    created: now(),
+    created: moment().unix(),
+    submittedBy: req.query ? req.query.username : null
```

Comment:
do we not have this as part of req.session?

Reply:
done